swagger/swaggerDoc.yaml

```diff
@@ -0,0 +1,849 @@
+      responses:
+        200:
+          description: "Status 200"
+    put:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"x\": 99,\n  \"y\": 87,\n  \"z\": 33,\n  \"radius\"\
+            : 55,\n  \"meta\": {\n    \"foo\": \"BAR\"\n  }\n}"
+      responses:
+        200:
+          description: "Status 200"
+    delete:
+      description: "IMHO: Bad Design - IntID is sth like an array of IDs?"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    parameters:
+    - name: "IntID"
+      in: "path"
+      required: true
+      type: "string"
+  /api/points/search:
+    post:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"x\": 23\n}"
+      responses:
+        200:
+          description: "Status 200"
+  /api/peripherals:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+            example: "[\n  {\n    \"id\": 4,\n    \"pin\": 526,\n    \"mode\": 1,\n\
+              \    \"label\": \"MyString\"\n  },\n  {\n    \"id\": 3,\n    \"pin\"\
+              : 239,\n    \"mode\": 1,\n    \"label\": \"MyString\"\n  }\n]"
+    post:
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"pin\": 13,\n  \"mode\": 0,\n  \"label\": \"LED\"\
+            \n}"
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+          examples:
+            application/json: "{\n  \"id\": 5,\n  \"pin\": 13,\n  \"mode\": 0,\n \
+              \ \"label\": \"LED\"\n}"
+    patch:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"pin\": 9\n}"
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+            example: "{\n  \"id\": 2,\n  \"pin\": 9,\n  \"mode\": 1,\n  \"label\"\
+              : \"MyString\"\n}"
+  /api/peripherals/{IntID}:
+    delete:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    parameters:
+    - name: "IntID"
+      in: "path"
+      required: true
+      type: "string"
+  /api/password_resets:
+    post:
+      description: "what happens if the emal-value has no \"@\"?"
```

Comment:
We have an email verification system in place- If you tried to change your email to something invalid, you would never get a verification email